File: X/clang/XRPL_UKL1/soal4.c
/*
Soal          : 4
Gambar        : B
Matrix kolom  : 5
*/

#include <stdio.h>

int main(){
    int A=6,
        B=3,
        kolom=5,
        u=A,
        s=B,
        total=0;
    printf("Deret Aritmatikanya adalah\n");
    for(int i=kolom;i>0;i--){ 
        for(int j=0;j<i;j++){
            printf("%d\t", u);
            u+=B;
            s+=u; 
        }
        printf("\n");
    }
    for(int i=kolom-1;i>0;i--){ 
        for(int j=kolom+1;j>0+i;j--){
            printf("%d\t", u);
            u+=B;
            s+=u; 
        }
        printf("\n");
    }
    total =s-u;
    printf("Jumlah Deret Aritmatikanya adalah %d\n", total);
}

File: X/typescript/soal4.ts
/*
Soal          : 4
Gambar        : b
Matrix kolom  : 5
*/

let a = 6,
  b = 3,
  kolom = 5;

let u = a,
  s = b;
console.log("Deret aritmatikanya adalah");

for (let i = kolom; i > 0; i--) {
  for (let j = 0; j < i; j++) {
    process.stdout.write(u + "\t");
    u += b;
    s += u;
  }
  console.log();
}

for (let i = kolom - 1; i > 0; i--) {
  for (let j = kolom + 1; j > 0 + i; j--) {
    process.stdout.write(u + "\t");
    u += b;
    s += u;
  }
  console.log();
}

let total = s - u;
console.log("Jumlah deret aritmatikanya adalah " + total);
